src/constants/url.ts

```diff
@@ -22,10 +22,12 @@ export const getOauthUrl = () => {
     const origin = window.location.origin;
     const hostname = window.location.hostname;
 
+    const existingAppId = LocalStorageUtils.getValue(LocalStorageConstants.configAppId);
+    const existingServerUrl = LocalStorageUtils.getValue(LocalStorageConstants.configServerURL);
     // since we don't have official app_id for staging,
     // we will use the red server with app_id=62019 for the staging-p2p.deriv.com for now
     // to fix the login issue
-    if (origin === URLConstants.derivP2pStaging) {
+    if (!/localhost/.test(origin) && (!existingAppId || !existingServerUrl)) {
```

Comment:
@farrah-deriv now for both staging and production it will set the url and app_id